vm/src/import.rs

```diff
@@ -63,28 +63,6 @@ pub fn import_module(
     Ok(module)
 }
 
-pub fn import(
```

Comment:
I think it is better to keep this function. Then call it from both the builtin `__import__` function, as well as from the import statement in frame.